### ISSUE TITLE
Crowdsource data normalization

### ISSUE THREAD

**Issue by mm12** [LAION-AI/Open-Assistant#1577]
Users will add have tendencies on how they score things (for example, some might put something helpful at 4, and only use 5 early) To account for this, in addition to asking multiple users, it should first normalize the scoring to how that user's response compares to others.

**Comment by andreaskoepf** [comment]
Yes, that's a good idea. @MattAlexMiracle already mentioned that he wants to analyze the user labeling and ranking structure. We will create an anonymized data export of user data (e.g. new random ids generated for each user).
